api/src/main/java/javax/enterprise/inject/spi/builder/AnnotatedTypeConfiguratorOld.java

```diff
@@ -0,0 +1,432 @@
+/*
+ * JBoss, Home of Professional Open Source
+ * Copyright 2016, Red Hat, Inc., and individual contributors
+ * by the @authors tag. See the copyright.txt in the distribution for a
+ * full listing of individual contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package javax.enterprise.inject.spi.builder;
+
+import javax.enterprise.inject.spi.AnnotatedConstructor;
+import javax.enterprise.inject.spi.AnnotatedField;
+import javax.enterprise.inject.spi.AnnotatedMethod;
+import javax.enterprise.inject.spi.AnnotatedParameter;
+import javax.enterprise.inject.spi.AnnotatedType;
+import javax.enterprise.inject.spi.ProcessAnnotatedType;
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Field;
+import java.lang.reflect.Method;
+import java.lang.reflect.Type;
+
+
+/**
+ * This API is an helper to configure a new {@link AnnotatedType} instance.
+ * The CDI container must provides an implementation of this interface.
+ *
+ * AnnotatedTypeConfigurator is not reusable.
+ *
+ * This configurator is not thread safe and shall not be used concurrently.
+ *
+ * @see javax.enterprise.inject.spi.BeforeBeanDiscovery#addAnnotatedType(String)
+ * @see javax.enterprise.inject.spi.AfterTypeDiscovery#addAnnotatedType(String)
+ * @see ProcessAnnotatedType#configureAnnotatedType()
+ * @param <T> the class represented by the configured AnnotatedType
+ * @author Antoine Sabot-Durand
+ * @since 2.0
+ */
+public interface AnnotatedTypeConfiguratorOld<T> {
```

Comment:
Remove this interface. It's just a "backup" - an old version of `AnnotatedTypeConfigurator` API.